core/templates/components/forms/custom-forms-directives/image-uploader.component.html

```diff
@@ -14,7 +14,7 @@
   </span>
   <label for="image-file-input" class="image-uploader-upload-label-button" *ngIf="!isBlogPostThumbnailUploader">{{ 'I18N_DIRECTIVES_UPLOAD_A_FILE' | translate }}</label>
   <!-- The onclick="this.value=null" attribute resets the value of the input so that uploading the same image twice in a row works. This fix is only needed for chrome. Reference: stackoverflow.com/q/12030686 -->
-  <input type="file" id="image-file-input" (click)="this.value = null"
+  <input type="file" id="image-file-input" (click)="this.imageInputRef.nativeElement.value = null"
```

Comment:
It was failing with some errors during angular build. Can't really remember the error message. happy to followup if someone wants to know.

Reply:
Checked on chrome and it works.

Reply:
Done!